ihatemoney/web.py

```diff
@@ -698,10 +699,34 @@ def delete_bill(bill_id):
     if not bill:
         return redirect(url_for(".list_bills"))
 
+    session["recently_deleted_bill"] = bill.to_json()
     db.session.delete(bill)
     db.session.commit()
-    flash(_("The bill has been deleted"))
 
+    url = url_for(".undo_delete_bill")
+    alert = (
+        'The bill has been deleted <a class="alert-link" href="'
+        + url
+        + '" id="undo"> undo </a>'
+    )
+    flash(Markup(alert))
```

Comment:
I find it weird to have this here, rather than in the templates. Maybe we should instead specify this in a template and display it if needed?

Reply:
sure, I will try doing it via template